app/src/main/java/com/amaze/filemanager/asynchronous/services/ftp/FtpService.java

```diff
@@ -95,10 +96,13 @@
     private static final String WIFI_AP_ADDRESS_PREFIX = "192.168.43.";
     private static final char[] KEYSTORE_PASSWORD = "vishal007".toCharArray();
 
-    // Service will (global) broadcast when server start/stop
-    static public final String ACTION_STARTED = "com.amaze.filemanager.services.ftpservice.FTPReceiver.FTPSERVER_STARTED";
-    static public final String ACTION_STOPPED = "com.amaze.filemanager.services.ftpservice.FTPReceiver.FTPSERVER_STOPPED";
-    static public final String ACTION_FAILEDTOSTART = "com.amaze.filemanager.services.ftpservice.FTPReceiver.FTPSERVER_FAILEDTOSTART";
+    // Service will broadcast via event bus when server start/stop
+    public enum FtpReceiverActions {
```

Comment:
Is there a reason to use an enum?

Reply:
We are emitting events within the app, we can choose the lightweight pojo path. enums are typed, pojo, and we don't need additional properties anyway, so I chose enums over static classes (per EventBus's example).